browser-test/src/applicant/questions/text.test.ts

```diff
@@ -162,6 +162,33 @@ test.describe(
       })
     })
 
+    test.describe('single text question without help text', () => {
+      const programName = 'Test program for single text without help text'
+
+      test.beforeEach(async ({page, adminQuestions, adminPrograms}) => {
```

Comment:
I originally wanted to group this test with the describe block above, however, the block above "single text question" looks like it's explicitly validating single text question state. I can add a test that adds a text question without help text but it would be against the spirit of the describe block. I added a separate describe block as a compromise but if you feel that the test would be better grouped with the first describe block even if it defies the mandate (we can't have a single text question with help text and without help text).

Reply:
Looks like after sheldon reformatted the tests there is a clearer pattern of separating the different testing conditions with describe, I'm going to resolve this conversation for now (looks like nick is at a conference this week) and will sync up with him via chat after he is back to see if he would prefer to reformat differently.